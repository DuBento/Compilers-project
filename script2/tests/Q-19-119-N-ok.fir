int *fir() -> 0 {
  float a = 3.14;
  float b = 1E3;
  float c = 12.34e-1;
  
  writeln a, ' ', b, ' ', c, ' ', a+b, ' ', b+c, ' ', a*b;
}
