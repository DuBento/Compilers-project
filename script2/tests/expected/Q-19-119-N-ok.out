3.14 1E3 1.234 1.00314E3 1.001234E3 3.14E3
